mindsdb/integrations/handlers/rag_handler/settings.py

```diff
@@ -280,7 +278,7 @@ class RAGBaseParameters(BaseModel):
     llm_params: Any
     vector_store_folder_name: str
     use_gpu: bool = False
-    embeddings_batch_size: int = MAX_EMBEDDINGS_BATCH_SIZE
+    embeddings_batch_size: int = MAX_EMBEDDINGS_BATCH_SIZE  # not used, leaving in place to prevent breaking changes
```

Comment:
leaving for now to allow for backward compatibility when loading users previously created models. We can remove at a later date